packages/suite-desktop/src-electron/modules/http-receiver.ts

```diff
@@ -49,6 +44,9 @@ const init: Module = ({ mainWindow }) => {
         logger.info('http-receiver', 'Stopping server (app quit)');
         httpReceiver.stop();
     });
+
+    // TODO check whether it's necessary to await httpReceiver.start()
+    return () => httpReceiver.getInfo();
```

Comment:
I'd say it should be awaited, yes. It seems like a possble source of raceconditions.

Reply:
true, i did catch it here :)
turns out `libs/http-receiver` class require a little bit more love :) 
currently it was fired up immediately from the constructor which was declared on top level of `http-receiver` module, now whole initialization it is moved into "load" function